test/metrics/lcom4/test_lcom4.py

```diff
@@ -32,41 +31,56 @@ class TestLCOM4(TestCase):
     pattern = LCOM4()
 
     def test_class_with_empty_method(self):
-        lcom4_val = self.pattern.value(Path(self.dir_path, 'EmptyMethod.java'))
+        name = 'EmptyMethod.java'
+        lcom4_val = self.pattern.value(Path(self.dir_path, name))
+        print(name)
         self.assertEqual(lcom4_val, 2)
 
-    def test_overloaded(self):
-        lcom4_val = self.pattern.value(Path(self.dir_path, 'Overloaded.java'))
-        lcom4_val2 = self.pattern.value(Path(self.dir_path, 'OverloadedDiffComp.java'))
-        self.assertEqual(lcom4_val, 1)
-        self.assertTrue(lcom4_val2, 2)
-
     def test_constructor(self):
-        lcom4_val = self.pattern.value(Path(self.dir_path, 'Constructor.java'))
+        name = 'Constructor.java'
+        lcom4_val = self.pattern.value(Path(self.dir_path, name))
+        print(name)
         self.assertEqual(lcom4_val, 1)
 
+    def test_scope_with_anonymous(self):
+        name = 'ScopeAnonymous.java'
+        lcom4_val = self.pattern.value(Path(self.dir_path, name))
+        print(name)
```

Comment:
why do u have print everywhere? You should remove it

Reply:
Done1